src/base.ts

```diff
@@ -29,6 +34,95 @@ export abstract class Base {
     this.baseUrl = "http://localhost:3000";
     this.disabledDataSettings = DisableDataSettings.None;
     this.enabled = true;
+    this.appDetails = {
+      platform_id: null,
+      app_version: null,
```

Comment:
we should rename this to `client_app_version` and the one below `server_app_version` so it's clearer